src/main/datalog_console/chrome/README.md

```diff
@@ -0,0 +1,36 @@
+# Datalog Chrome Extension
```

Comment:
I love the docs, but I don’t love that they’re duplicated in two READMEs. Can you choose one and remove the other? Maybe link to this readme from the main readme so the main readme can focus on consuming datalog-console rather than contributing to it.

Reply:
I missed this when doing cleanup. I think we just keep them all at the top level.